mobile-ui/src/main/java/com/playone/mobile/ui/model/SignUpViewModel.kt

```diff
@@ -0,0 +1,31 @@
+package com.playone.mobile.ui.model
+
+import android.arch.lifecycle.ViewModel
+import android.arch.lifecycle.ViewModelProvider
+import com.playone.mobile.presentation.onBoarding.LoginPlayoneContract
+
+class SignUpViewModel(presenter: LoginPlayoneContract.Presenter)
+    : LoginViewModel(presenter) {
+
+    init {
+        presenter.setView(this)
```

Comment:
Parent class `LoginViewModel ` will init first, child class doesn't have to init again.